apps/mull-ui/src/app/pages/messages/event-chat/event-chat.tsx

```diff
@@ -99,7 +100,11 @@ export const EventChat = ({ history, channelName, restrictChatInput }: EventChat
     },
 
     validationSchema: Yup.object({
-      message: file ? Yup.string().optional() : Yup.string().required(),
+      message: file
+        ? Yup.string().optional()
+        : Yup.string()
+            .required()
+            .test('no-whitespace', "Message can't be empty", (value) => !/^\s*$/.test(value)),
```

Comment:
Message isn't really empty since it does have a space character. Not having any characters will also show the user "message is a required field" maybe just not have any warnings for the user

Reply:
"maybe just not have any warnings for the user", well no they can't send a message that is empty or just has whitespace. I'll adjust the error message

Reply:
Well because now there's two different error message that trying to say the same thing.

When you type a space, the message isn't empty.

I don't think showing an error message for sending an empty message or a space.

Reply:
Sure, we could have it just not do anything. This actually also solved another issue where the error message blocks off messages. will do